.ci/jobs/apm-it-ec.yml

```diff
@@ -0,0 +1,44 @@
+---
+- job:
+    name: apm-it-ec
```

Comment:
What do you think if we create a new folder for allocating all the APM-ITs?

Reply:
no have a folder was a request on the past, but agree we can start to think to create a folder for them